### PR TITLE
Keep sticker handle size while zoom in/out image/text

### PR DESCRIPTION
**Pull request by tiendung717**

### CODE REVIEW

**Comment by lucianocheng** [comment]
Ran some tests. Sticker handles staying the same size looks good 👍 .

Two things:

1. When I take a handle and drag, and let go, only the handle I was dragging becomes visible. When I let go, I believe all four handles should become visible. This is so the user can always grab a handle, but while dragging the 3 "unused" handles don't obstruct the picture.

2. I think the handles should be a bit bigger, maybe 50% for their fixed size.

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/HandleView.kt [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import android.annotation.SuppressLint
+import android.content.Context
+import android.util.AttributeSet
+import android.util.Log
+import android.view.Gravity
+import android.widget.FrameLayout
+import android.widget.ImageView
+
+@SuppressLint("AppCompatCustomView")
+class HandleView(context: Context, attributeSet: AttributeSet) : ImageView(context, attributeSet),
```
Comment (lucianocheng) [inline photoeditor/src/main/java/ja/burhanrashid52/photoeditor/HandleView.kt:12]: For new classes: Please leave a brief description at the top of each class describing what they do. Thanks.

**Review by lucianocheng — APPROVED** [review]

File: photoeditor/src/main/java/ja/burhanrashid52/photoeditor/HandleView.kt (hunk 1 above)
Comment (tiendung717) [reply photoeditor/src/main/java/ja/burhanrashid52/photoeditor/HandleView.kt:12]: Yes, I added some comments for HandlerView and BorderView classes

**Comment by tiendung717** [comment]
> Ran some tests. Sticker handles staying the same size looks good +1 .
> 
> Two things:
> 
> 1. When I take a handle and drag, and let go, only the handle I was dragging becomes visible. When I let go, I believe all four handles should become visible. This is so the user can always grab a handle, but while dragging the 3 "unused" handles don't obstruct the picture.
> 2. I think the handles should be a bit bigger, maybe 50% for their fixed size.
> 3. It looks like a lot of logic was moved from the ThugLife library to here. In general, I want to keep this library as close to the open source implementation as possible, and anything that isn't in there now we should assume we will eventually need to up merge to the open source repository.. If something doesn't have to be in here, I'd rather expose the ability to add it in a small way in this library, and have the meat of the code in ThugLife.
> 
> Just to give an example: For the handles, if we can get access to the handle views, I'd rather do that then have the handle view logic in Thug Life.
> 
> It's also possible I'm misunderstanding the changes so any clarity would be great. Thanks.

**Comment by lucianocheng** [comment]
Got it. Thanks for the explanation. Left some thoughts below but overall approved 👍 

> Create custom views for HandleView and BorderView, and handle size change by adjusting size, margin inside these custom views.

HandleView 100% makes sense to be in the SDK.

BorderView may not make sense, but this is for a weird reason. We actually do a lot of custom border work in our fork of PhotoEditorSDK. A lot of this may be unnecessary and I may remove from our fork. However, given where our fork is now, it is fine for now.

> If you consider about the conflict when merging with the original SDK, I don't think it's a problem, because the changes I made it in separate classes and there will be easy for merging without any conflict.

Agreed this will not conflict with existing code in the original SDK. However like any large open source project, unfortunately this is not the only condition for contributing.

In the case of PhotoEditorSDK, things like introducing new callbacks, hook points, "API" changes, or additional logic has ancillary requirements. These include feature flags, tests, "broad project consistency", review delay, preserving existing functionality even things like as bugs, etc. These all take time and often require negotiation to complete. It once took me 16 months to get a feature change through review, specifically adding a "currently selected" callback.

These issues are not present in the kotlin transition because that review is keeping 100% functional equivalency. Things like bug fixes (e.g., file permission issues), are also easy. However new features (even exposing callbacks) are tricky.

Anyway, this is OK for now, just wanted to give you some context. Thanks!

**Pull request merged**